internal/config/config.go

```diff
@@ -108,7 +109,8 @@ func (c Config) String() string {
 	fmt.Fprintf(&b, "%s: %d\n", JOB_RECEIVER_RECEPTOR_PROXY_PORT, c.JobReceiverReceptorProxyPort)
 	fmt.Fprintf(&b, "%s: %s\n", JOB_RECEIVER_RECEPTOR_PROXY_TIMEOUT, c.JobReceiverReceptorProxyTimeout)
 	fmt.Fprintf(&b, "%s: %s\n", GATEWAY_CONNECTION_REGISTRAR_IMPL, c.GatewayConnectionRegistrarImpl)
-	fmt.Fprintf(&b, "%s: %s\n", GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_DELAY, c.GatewayActiveConnectionRegistrarPollDelay)
+	fmt.Fprintf(&b, "%s: %s\n", GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_MIN_DELAY, string(c.GatewayActiveConnectionRegistrarPollMinDelay))
+	fmt.Fprintf(&b, "%s: %s\n", GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_MAX_DELAY, string(c.GatewayActiveConnectionRegistrarPollMaxDelay))
```

Comment:
If you change the second %s to a %d, then it should work with needing to convert GatewayActiveConnectionRegistrarPollMinDelay and GatewayActiveConnectionRegistrarPollMaxDelay to strings
```
       fmt.Fprintf(&b, "%s: %d\n", GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_MIN_DELAY, c.GatewayActiveConnectionRegistrarPollMinDelay)
       fmt.Fprintf(&b, "%s: %d\n", GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_MAX_DELAY, c.GatewayActiveConnectionRegistrarPollMaxDelay)
```